install/docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.6
+
+LABEL description="Airsonic is a free, web-based media streamer, providing ubiquitious access to your music." \
+      url="https://github.com/airsonic/airsonic"
+
+ENV AIRSONIC_PORT=4040 AIRSONIC_DIR=/airsonic
+
+WORKDIR $AIRSONIC_DIR
+
+RUN apk --no-cache add \
+    ffmpeg \
+    lame \
+    bash \
+    libressl \
+    ca-certificates \
+    tini \
+    openjdk8-jre
+
+COPY run.sh /usr/local/bin/run.sh
+
+RUN chmod +x /usr/local/bin/run.sh
+
+COPY target/dependency/airsonic-main-6.3-SNAPSHOT.war airsonic.war
```

Comment:
Just my sidenote: instead of hardcoding version of war here better to have something like this:

```
ARG AIRSONIC_VERSION
COPY target/dependency/airsonic-main-${AIRSONIC_VERSION}.war airsonic.war
```
and set this version via plugin, see plugin [MR](https://github.com/spotify/dockerfile-maven/pull/41)

Also this `AIRSONIC_VERSION` can be used in `LABEL`